src/main/java/wolox/training/model/Book.java

```diff
@@ -0,0 +1,55 @@
+package wolox.training.model;
+
+import com.sun.istack.NotNull;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import lombok.Data;
+
+@Data
+@Entity
+@Table(name = "book")
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
While this you use is fine, it would be better if this is used:
@GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "BOOK_SEQ")
@SequenceGenerator(name = "BOOK_SEQ", sequenceName = "BOOK_SEQ")

because PostgreSQL uses a global incremental id, shared by all tables :)